src/plugins/Debug.ts

```diff
@@ -0,0 +1,85 @@
+import insertElement from '../utils/insertElement';
```

Comment:
This file no longer exists in master by design. Maybe some bad rebase stuff going on in here -- or just in need of one